template.json

```diff
@@ -468,8 +468,9 @@
         "injuries": 0,
         "max": 0
       },
+
       "ap": {
-        "value": 5
+        "current": 5
       },
```

Comment:
You cannot just make a change to template.json and expect it to work. 

Everyone that updates their foundry to this version all their trainer actors would now have an `ap.value` field set to whatever it was set and a new field created `ap.current` with value 5. A table that left in the middle of a session would suddenly have all their AP reset to 5.

Renaming a value like this should only be done if it's actually necessary, and then a migration script needs to be written as well.

Please change it back to `value` as well as every reference from `current` back to `value`, or ping me on discord if you have a good reason for changing it 😉